### ISSUE TITLE
verbesserte Suche

### ISSUE THREAD

**Issue by Flexmaen** [gschwann/Baugeschichte#42]
Die API zur Suche würde bessere Möglichkeiten bieten als derzeit in der App vorhanden.

**Comment by Flexmaen** [comment]
Eigentlich funktioniert die Suche derzeit überhaupt nicht... Bitte testen.

**Comment by gschwann** [comment]
Funktioniert bei mir unter Linux und im Android Simulator ...

**Comment by Flexmaen** [comment]
Nein, da fehlt noch einiges, aber mir ist jetzt klar, dass Du beim Testen wahrscheinlich nicht alle möglichen Fälle testest.

Suche mach:
"meran" => Suchergebnis lässt sich nicht öffnen (weil nicht genauer augeschlüsselt)
"grazwiki" => Suchergebnis ist keine Adresse, die in der App darstellebar ist
"kunsthaus" => da kommt seltsamerweise kein Suchergebnis, obwohl auf der Webseite der Südtirolerplatz 2 kommt.

**Comment by Flexmaen** [comment]
Ev. wenn kein Suchergebnis kommt dann eine Nominatim-Afrage ( http://wiki.openstreetmap.org/wiki/Nominatim ) und einfach zur Position auf der Karte springen oder so.
